### PR TITLE
DOC: random: Fix a comment and example in the multivariate_normal docstring

### PR DESCRIPTION
**Pull request by WarrenWeckesser**
The covariance matrix is the identity, so the standard deviation of
each component is 1.0.

### CODE REVIEW

**Comment by bashtage** [comment]
I wonder if the first part was originally supposed to be a mean across the 9 elements generated so that the standard deviation of the mean would be 1/3, and so 2*1/3 is around 0.6.

**Comment by WarrenWeckesser** [comment]
@bashtage, maybe, but either way, I don't find the example to be very illustrative.  So, I replaced that short example with a scatter plot of a sample from the bivariate normal distribution.

**Comment by mattip** [comment]
The rendered document is [here](https://23751-908607-gh.circle-artifacts.com/0/doc/build/html/reference/random/generated/numpy.random.Generator.multivariate_normal.html?highlight=scatter%20plot). Could you add a little more descriptive text and an alt text for accessibility? What exactly is the contribution of the plot: what are we meant to be looking at, how does it enhance the understanding of the function parameters?

**Comment by WarrenWeckesser** [comment]
Thanks @mattip, I'll add some description.

> ...  and an alt text for accessibility?

How do we do that with a plot generated by Sphinx?  I can't find any other plots in our docs that have a useful alt text.  It looks like the `alt` attribute is always the file name generated by Sphinx  (e.g. see the plots in [`Generator.power` docstring](https://23751-908607-gh.circle-artifacts.com/0/doc/build/html/reference/random/generated/numpy.random.Generator.power.html)).

**Comment by WarrenWeckesser** [comment]
@mattip, I pushed an update; check out the [rendered version](https://23772-908607-gh.circle-artifacts.com/0/doc/build/html/reference/random/generated/numpy.random.Generator.multivariate_normal.html#numpy.random.Generator.multivariate_normal).

**Comment by mattip** [comment]
> How do we do that with a plot generated by Sphinx? 

I guess we can leave that for a future PR

> I pushed an update

Thanks, much clearer now. @bashtage thoughts?

File: numpy/random/_generator.pyx [hunk 1]
```diff
@@ -3572,13 +3572,30 @@ cdef class Generator:
         >>> y.shape
         (3, 3, 2)
 
-        Here we generate a scatter plot of 800 samples from the bivariate
-        normal distribution with mean (0, 0) and covariance matrix
-        [[6, -3], [-3, 3.5]].
+        Here we generate 800 samples from the bivariate normal distribution
+        with mean (0, 0) and covariance matrix [[6, -3], [-3, 3.5]].  The
```
Comment (bashtage) [inline numpy/random/_generator.pyx:3576]: should you use [ ] consitently, e.g. ,replace (0, 0) with [0, 0]

File: numpy/random/_generator.pyx [hunk 2]
```diff
@@ -3572,13 +3572,30 @@ cdef class Generator:
...
+        expected variances of the first and second components of the sample
+        are 6 and 3.5, respectively, and the expected correlation
```
Comment (bashtage) [inline numpy/random/_generator.pyx:3578]: Probably not a big deal, but the expected correlation is somewhat closer to 0 than the population correlation, due to finite sample bias in the estimator.

File: numpy/random/mtrand.pyx [hunk 3]
```diff
@@ -4079,13 +4079,30 @@ cdef class RandomState:
         >>> x.shape
         (3, 3, 2)
 
-        Here we generate a scatter plot of 800 samples from the bivariate
-        normal distribution with mean (0, 0) and covariance matrix
-        [[6, -3], [-3, 3.5]].
+        Here we generate 800 samples from the bivariate normal distribution
```
Comment (bashtage) [inline numpy/random/mtrand.pyx:4082]: Ideally any changes made to the above are replicated here.

**Review by bashtage — COMMENTED** [review]
Some thoughts. Nothing that is essential.

File: numpy/random/_generator.pyx (hunk 2 above)
Comment (WarrenWeckesser) [reply numpy/random/_generator.pyx:3578]: It looks like the magnitude of the first term in the [correction](https://en.wikipedia.org/wiki/Pearson_correlation_coefficient#Adjusted_correlation_coefficient) would be ≈ 2.3e-4, so probably not worth mentioning in a simple example like this.

File: numpy/random/_generator.pyx (hunk 1 above)
Comment (WarrenWeckesser) [reply numpy/random/_generator.pyx]: Done.

File: numpy/random/mtrand.pyx (hunk 3 above)
Comment (WarrenWeckesser) [reply numpy/random/mtrand.pyx:4082]: Done.

**Comment by mattip** [comment]
Thanks @WarrenWeckesser

**Pull request merged**